src/application/service/auth.service.spec.ts

```diff
@@ -57,89 +53,63 @@ describe('AuthService', () => {
+      queryPortMock.getUserBySnsId.mockResolvedValueOnce(null);
 
-      const credential = userCredentialMockData;
-      queryPortMock.getUserForLogin.mockResolvedValue(credential);
+      commandPortMock.createUser.mockResolvedValue(2);
+      jwtService.sign.mockReturnValueOnce('accessToken').mockReturnValueOnce('refreshToken');
 
-      const command = { email: credential.email, password: credential.password };
+      const newSnsId = 'sns-new-456';
+
+      const command: SocialLoginCommand = {
+        snsId: newSnsId,
+        nickname: '새로운 유저',
+        profileImageUrl: 'http://new.image.jpg',
+      };
 
       // when
-      const result = await service.login(command);
+      const result = await service.loginWithSocial(command);
 
       // then
       expect(result).toEqual({ accessToken: 'accessToken', refreshToken: 'refreshToken' });
-      expect(queryPortMock.getUserForLogin).toHaveBeenCalledTimes(1);
-      expect(queryPortMock.getUserForLogin).toHaveBeenCalledWith(command.email);
-      expect(bcrypt.compare).toHaveBeenCalledTimes(1);
-      expect(bcrypt.compare).toHaveBeenCalledWith(command.password, credential.password);
-    });
 
-    it('로그인하려는 사용자가 존재하지 않으면 로그인에 실패한다', async () => {
-      // given
-      jest.spyOn(queryPortMock, 'getUserForLogin').mockRejectedValue(new UserNotFoundException());
-
-      const credential = userCredentialMockData;
-      const command = { email: credential.email, password: credential.password };
-
-      // when & then
-      await expect(service.login(command)).rejects.toThrow(new UserNotFoundException());
-      expect(queryPortMock.getUserForLogin).toHaveBeenCalledTimes(1);
-      expect(queryPortMock.getUserForLogin).toHaveBeenCalledWith(command.email);
-      expect(bcrypt.compare).not.toHaveBeenCalled();
-      expect(jwtService.sign).not.toHaveBeenCalled();
-    });
-
-    it('비밀번호가 일치하지 않으면 로그인에 실패한다', async () => {
-      // given
-      (bcrypt.compare as jest.Mock).mockResolvedValue(false);
-
-      const credential = userCredentialMockData;
-      queryPortMock.getUserForLogin.mockResolvedValue(credential);
-
-      const command = { email: credential.email, password: 'wrongPassword' };
-
-      // when & then
-      await expect(service.login(command)).rejects.toThrow(new InvalidPasswordException());
-      expect(queryPortMock.getUserForLogin).toHaveBeenCalledTimes(1);
-      expect(queryPortMock.getUserForLogin).toHaveBeenCalledWith(command.email);
-      expect(bcrypt.compare).toHaveBeenCalledTimes(1);
-      expect(bcrypt.compare).toHaveBeenCalledWith(command.password, credential.password);
-      expect(jwtService.sign).not.toHaveBeenCalled();
+      expect(queryPortMock.getUserBySnsId).toHaveBeenCalledWith(command.snsId);
+      expect(commandPortMock.createUser).toHaveBeenCalledWith(
+        expect.objectContaining({
+          snsId: command.snsId,
+          nickname: command.nickname,
+          profileImageUrl: command.profileImageUrl,
+        }),
+      );
```

Comment:
없어도 괜찮을 것 같은데 한번 확인 부탁드릴게요

```suggestion
{
          snsId: command.snsId,
          nickname: command.nickname,
          profileImageUrl: command.profileImageUrl,
        });
```

Reply:
필드가 추후에 좀 변경될 수도 있을거 같아서 그렇게 작성했는데 어차피 3개 필드 고정으로 받을거 같아서 그렇게 수정하겠습니다!